src/components/CanvasEditor/Footer/Help/Help.scss

```diff
@@ -33,21 +34,41 @@
   }
 
   &__shortcuts {
-    border: 1px solid var(--light-border-color);
-    border-radius: var(--default-border-radius);
+    display: flex;
```

Comment:
Let's not remove border & border-radius & I think we don't need flex ,flex-direction: column. We can remove that too .